src/apps/MultitouchExtension/src/FingerManager.swift

```diff
@@ -80,8 +90,9 @@ class FingerManager: ObservableObject {
     //
 
     for s in states {
-      if s.device == device && s.frame != frame && s.touchedPhysically {
+      if s.device == device && s.frame != frame && (s.touchedPhysically && !s.palmed) {
```

Comment:
Looking at this logic here, I'm not sure I changed this expression correctly.

I don't really understand the purpose of this block to be honest. If the frame changes (from my testing, isn't frame just monotonically increasing?), we disable `s.touchedPhysically` and start the untouched delay task? Why is that?

Reply:
It is simpler to treat it as a simple flag in the same way as `ignored`.
I updated the logic in #3564.
https://github.com/pqrs-org/Karabiner-Elements/pull/3564/files?w=1